### PR TITLE
Mention usage of Git LFS in Changelog

### PR DESCRIPTION
**Pull request by sgiehl**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ The Product Changelog at **[piwik.org/changelog](http://piwik.org/changelog)** l
 ### New APIs
 * The Piwik JavaScript tracker has a new method `trackRequest` that allows you to send any tracking parameters to Piwik. For example  `_paq.push(['trackRequest', 'te=foo&bar=baz'])`
 
+### Breaking Changes
```
Comment (mattab) [inline CHANGELOG.md]: I think should be "Internal change" instead as it's only breaking for developers running tests

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,10 @@ The Product Changelog at **[piwik.org/changelog](http://piwik.org/changelog)** l
...
+* Expected screenshots for UI tests are now stored using Git LFS instead of a submodule. Running, creating or updating UI tests will require Git LFS to be installed.
+Therefor the folder containing expected screenshots was renamed from `expected-ui-screenshots` to `expected-screenshots`. The UI-Test-Runner is now able to handle both names.
```
Comment (mattab) [inline CHANGELOG.md]: Therefor the folder containing ->  The folder containing

Comment (mattab) [inline CHANGELOG.md]: The UI-Test-Runner is now able to handle both names. -> maybe not needed?

Comment (sgiehl) [inline CHANGELOG.md]: The change of the directory also effects UI tests nested in plugins. In order not to break existing plugin tests I implemented the possibility to handle both directories.

**Pull request merged**